sunpy/net/attr.py

```diff
@@ -60,6 +87,43 @@ def collides(self, other):
     def __eq__(self, other):
         return dict(vars(self)) == dict(vars(other))
 
+    @classmethod
+    def update_values(cls, adict):
+        """
+        This is how clients will register their `Attrs` with the system.
+        The input should be a dictionary.
+        Each key should be a type of the attrs you want.
+        The value should be a list of pairs.
+        The pair should be a tuple of (Name, Description).
+        If you do not want to add a description, you can put `"N/A"`.
+        We sanitize the name and it becomes an attribute on the attrs class.
+        # Do we want to have them use type?
+
+        We have an example here for an Instrument Class.
+
+        Example
+        -------
+        >>> from sunpy.net import attr, attrs
+        >>> attr.Attr.update_values({type(attrs.Instrument): [('AIA', 'AIA is in Space.'), ('HMI', 'HMI is next to AIA.')]})
+
+        """
+        for k, v in adict.items():
+            if isiterable(v) and not isinstance(v, str):
+                for pair in v:
+                    if len(pair) != 2:
+                        raise Warning
+                    else:
+                        # Sanity name, we remove all special characters and make it all lower case
+                        name = ''.join(char for char in pair[0] if char.isalnum()).lower()
+                        if keyword.iskeyword(name) or not name.isidentifier():
+                            raise Warning
```

Comment:
rather than just spewing here could we append or prepend something to make it not a keyword / identifier anymore?